### PR TITLE
chore(vscode): request code ownership on vscode

### PR DESCRIPTION
**Pull request by alestiago**
## Status

**READY**

## Description

Requesting code ownership on vscode extension. Mainly because I want to get the dependabot Pull Requests reviewed and merged.

I'm using the syntax shown [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file).

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [X] 🗑️ Chore

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Every request must be reviewed and accepted by:
 
-*	@felangel @renancaraujo @wolfenrain @erickzanardo
+- @felangel @renancaraujo @wolfenrain @erickzanardo
```
Comment (alestiago) [inline .github/CODEOWNERS:3]:
```suggestion
* @felangel @renancaraujo @wolfenrain @erickzanardo
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,3 +1,5 @@
 # Every request must be reviewed and accepted by:
 
-*	@felangel @renancaraujo @wolfenrain @erickzanardo
+* @felangel @renancaraujo @wolfenrain @erickzanardo
+
+/extension/vscode/ @alestiago
```
Comment (alestiago) [inline .github/CODEOWNERS:5]:
```suggestion
/extensions/vscode/ @alestiago
```

**Review by wolfenrain — DISMISSED** [review]

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,3 +1,5 @@
 # Every request must be reviewed and accepted by:
 
-*	@felangel @renancaraujo @wolfenrain @erickzanardo
+* @felangel @renancaraujo @wolfenrain @erickzanardo
+
+/extensions/vscode/ @alestiago
```
Comment (renancaraujo) [inline .github/CODEOWNERS:5]:
```suggestion
* @felangel @renancaraujo @wolfenrain @erickzanardo @alestiago
```

**Review by renancaraujo — CHANGES_REQUESTED** [review]
cmon man

**Pull request merged**